group/views.py

```diff
@@ -20,3 +23,100 @@ def create_group(request):
             return Response(serializer.data, status=status.HTTP_201_CREATED)
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
     return None
+
+
+@api_view(['PATCH'])
+@permission_classes([IsAuthenticated])
+def add_users(request):
+    """Add user(s) to a group"""
+    if request.method == 'PATCH':
+        data = request.data.copy()
+        group_id = request.data.get("group_id")
+
+        data.pop('creator', None)
+        data.pop('name', None)
+        try:
+            group = Group.objects.get(id=group_id)
+            if request.user != group.creator:
+                return Response({
+                                "error": "Only group creator can add members"},
+                                status=status.HTTP_403_FORBIDDEN)
+            errors = []
+            for member in group.members.all():
```

Comment:
This logic looks abit off, you are looping through the members in the group, and checking if a member is not in the request body members, and then appending that member to the request object. Why? I was assuming, you are adding new members to the group. So I would expect that you simply update the the members field with the new items, as opposed to gettting the old items and adding them to the new one. So this looks irrelevant for me.